packages/@cdktf/provider-generator/lib/get/generator/emitter/attributes-emitter.ts

```diff
@@ -295,6 +298,13 @@ export class AttributesEmitter {
           )}ToTerraform(${varReference}.internalValue),`
         );
         break;
+      case type.isComplex && !type.struct?.isClass && !type.isSingleItem:
+        this.code.line(
+          `${att.terraformName}: ${defaultCheck}${downcaseFirst(
+            type.struct!.name
+          )}ToTerraform(${varReference}),`
+        );
+        break;
```

Comment:
What is this new case handling? (Just out of curiosity)

Reply:
It's where a non-computed attribute is specified as an object. From what I've seen it is quite rare, but not having it did break the [Kubernetes provider manifest resource wait_for attribute](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/manifest#using-wait_for-to-block-create-and-update-calls).

Reply:
Ah, nice. Thanks!